Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_UITests.xcscheme

```diff
@@ -264,6 +264,7 @@
       buildConfiguration = "Fennec_Enterprise"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = ""
```

Comment:
unstage

Reply:
I think this is std thing xcode 9 adds now for all schemes when modified